app/views/api/application_plans/index.html.slim

```diff
@@ -1,9 +1,11 @@
-h1 Application plans
+h1 Application Plans
```

Comment:
Wait a second, this looks contradictory. My understanding is that only first letter is capitalized. That's what we have been doing, like with Mapping rules.

Reply:
Application Plans is correct usage in this context. https://www.patternfly.org/v4/ux-writing/capitalization/#capitalization-across-red-hat-uis